Formula/zig.rb

```diff
@@ -0,0 +1,27 @@
+class Zig < Formula
+  desc "Programming language designed for robustness, optimality, and clarity"
+  homepage "http://ziglang.org/"
+  url "https://s3.amazonaws.com/ziglang.org/download/0.1.1/zig-0.1.1.tar.xz"
+  sha256 "a160d14aecebead2f8c574b0dd687cdfecc53d149d521a17b89b629d8b558c94"
+
+  depends_on "cmake" => :build
+  depends_on "llvm" => :build
```

Comment:
zig links to LLVM's libc++ (`/usr/local/opt/llvm/lib/libc++.1.dylib`). So this dependency is necessary at runtime too. Remove the `=> :build`.

Reply:
Done, thanks for sussing that out!